lib/__tests__/wrappers/ModalWrapper.test.js

```diff
@@ -10,7 +10,7 @@ describe('ModalWrapper', () => {
   });
 
   it('Should renders', () => {
-    console.log(component.debug()); // TODO REMOVE ME
+    // console.log(component.debug());
```

Comment:
why comment and not just remove the line? it's not pretty 😅

Reply:
Oh, thats just really helpfull to just uncomment line and have an ability to review component)